interface-spec.md

```diff
@@ -28,6 +28,11 @@ TODO: read/write or read-only?
 
 TODO: to what extent should we use typed signatures (`.equals(Node other)`) versus actual JavaScript signatures (`.equals(other)`). The benefit of typed signatures is that you see the type inline; the drawback is that it is more specific than JavaScript itself. For ease of use, JavaScript might be preferred, specifying types in the explanation (or jsdoc-style).
 
+**Methods not specified:**
+
+- `.toString()` is not specified, therefore the results may vary for each implementation.
+The property `.value` and method `.toCanonical()` should be used to access the value of a term.
+
```

Comment:
I agree on the "methods not specified". How many of them will we have? I expect it will be only `.toString()`. But people will search for it. Beside your proposal for the `.toCanonical()` I would add the `.toString()` method to the list of methods with a text like: "not specified, see `.toCanonical()`". If it's only on the `.toCanonical()` text, it's a little bit hidden. We should do this only for the special case of `.toString()`!